src/AbstractDifferentiation.jl

```diff
@@ -127,6 +214,14 @@ function value_gradient_and_hessian(ab::HigherOrderBackend, f, x)
     return value, (grads,), hess
 end
 
+"""
+    AD.pushforward_function(ab::AD.AbstractBackend, f, xs...)
+    
+Returns the pushforward function `pf_f` of the function `f` at the inputs `xs`. 
+    
+`pf_f` is a function that accepts the tangents `vs` as input which is a tuple of length equal to the length of the tuple `xs`.
+If `f` has a single input, `pf_f` can also accept a single input instead of a 1-tuple.
```

Comment:
I think it reads a bit easier without underscores (and also for gradients, Jacobians, hessians, etc. we don't explicitly mention `f` in the notation).
```suggestion
Return the pushforward function `pf` of the function `f` at the inputs `xs` using backend `ab`. 
    
The pushfoward function `pf` accepts as input a `Tuple` of tangents, one for each element in `xs`.
If `xs` consists of a single element, `pf` can also accept a single tangent instead of a 1-tuple.
```